srcpkgs/noisetorch/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'noisetorch'
+pkgname=noisetorch
+version=0.11.3
+revision=1
+wrksrc=NoiseTorch-${version}
+build_style=go
+go_ldflags="-X main.version=${version} -X main.distribution=voidlinux"
+go_import_path="."
+depends="polkit"
+short_desc="Real-time microphone noise suppression on Linux"
+maintainer="Soc Virnyl S. Estela <renegan.ronin@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/lawl/NoiseTorch"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=b67155211278affc72f15d0ded9da560524e7bb0659012f9d41e69b531b9265c
+nopie=yes
```

Comment:
why? can you leave a comment?

Reply:
im going to delete that line. that was from the old template i have before.